doc/user-guide/user-guide.rst

```diff
@@ -45,30 +45,56 @@ CCArchiver User Guide
 #######################
 
 .. Update links and labels below
-.. image:: https://img.shields.io/badge/GitHub-ts_athexapod-green.svg
-    :target: https://github.com/lsst-ts/ts_athexapod
-.. image:: https://img.shields.io/badge/Jenkins-ts_athexapod-green.svg
-    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_athexapod/
-.. image:: https://img.shields.io/badge/Jira-ts_athexapod-green.svg
-    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_athexapod
-.. image:: https://img.shields.io/badge/ts_xml-ATHexapod-green.svg
-    :target: https://ts-xml.lsst.io/sal_interfaces/ATHexapod.html
+.. image:: https://img.shields.io/badge/GitHub-dm_ccarchiver-green.svg
+    :target: https://github.com/lsst-dm/dm_CCArchiver
+.. image:: https://img.shields.io/badge/Jenkins-dm_CCArchiver-green.svg
+       :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/dm_CCArchiver/
+.. image:: https://img.shields.io/badge/Jira-dm_ccarchiver-green.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+dm_ccarchiver
+.. image:: https://img.shields.io/badge/ts_xml-CCArchiver-green.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/CCArchiver.html
 
 
-[This area should give an introduction to users, but at greater depth than the overview on the higher-level page. Some repetition is fine.
-Discussion of high-level control classes (if applicable), primary use cases, links to any useful documentation etc can be included.
-Reference to configurations may also be worthwhile]
+The Archiver interacts with the Archive Controller and the Forwarder to 
+coordinate triggering image retrieval and storage. It receives commands 
+and events from the SAL DDS messaging system.  This includes commands to 
+change state and events from the Camera and Header Service.  
 
-CCArchiver Interface
-======================
+The Archiver also interacts with the Redis key/value store to hold state 
+information, and the RabbitMQ broker to send/receive messages from the 
+Controller, Forwarder, and Observatory Operations Data Service (OODS).
 
-[This area should link to the XML, then discuss the primary commands/events/telemetry that are expected to be used.
-A full explanation of all possible commands/events/telemetry is beyond the scope of this document.
-However, the description in the XML should be verified to be adequately populated such that a user can find the deeper information if required.]
+The Archive Controller handles operations for images. It sets up storage
+locations that the Forwarder uses and creates hard links for incoming 
+files which are later used by the OODS and the Data Backbone.
```

Comment:
So one thing I've not yet encountered is discussion of data transfer between the site and NCSA. I realize this isn't part of the archiver itself, but it's worth a sentence (probably in a few places) to explain that dividing line.
If there are references to the long-haul network you can link to that'd be great.... but i'd be surprised if they exist.
In this sentence you are talking about a local OODS and Data Backbone, yes? Not the one at NCSA?

Reply:
Yes, local OODS and local Data Backbone endpoint.  I added some information that describes this.